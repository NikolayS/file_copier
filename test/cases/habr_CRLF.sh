#!/bin/bash

# to run this scenario, COPIERSERVICE should be defined (`export COPIERSERVICE=https://your.copier.host.name`)

http GET "$COPIERSERVICE/?wh=1&md5=1&src=https://habrastorage.org/getpro/habr/post_images/fe7/5a1/3bb/fe75a13bbbd36c015b34a41bb0df990b.jpg" -h

