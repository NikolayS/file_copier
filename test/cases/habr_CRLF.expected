HTTP/1.1 200 OK
Server: nginx
Content-Type: text/html; charset=UTF-8
Transfer-Encoding: chunked
Connection: keep-alive
X-File-Copier-Md5: 54a7d8d750eb2ac3e16717088ab5a060
X-File-Copier-Img-Width: 603
X-File-Copier-Img-Height: 452
X-File-Copier-Size: 42296
X-Location: /data/96/24/37/f0/962437f08d590b170d71c591b5fd3f7c88cc957302b13491a765df2ae1834090.jpg
Access-Control-Expose-Headers: X-File-Copier-Size, X-Location, X-File-Copier-Img-Height, X-File-Copier-Img-Width, X-File-Copier-Md5
Content-Encoding: gzip

